packages/types/src/market.ts

```diff
@@ -40,3 +40,43 @@ export type FindAllMarketArgs = {
 export type MarketCapResult = {
   [k: string]: MarketData
 }
+
+/**
+ * stackedQ: We need the list in the frontend,
+ * and there's no way to convert an union type to an array,
+ * so the list is defined as an array of strings,
+ * but SupportedFiatCurrencies is an union of strings
+ */
+export const SupportedFiatCurrenciesList = [
```

Comment:
we don't actually need this list - if we define a string union, the compiler is smart enough to autocomplete which strings are valid types

Reply:
This list is used to render the supported fiat currencies in the "currency" section of the settings modal. I couldn't find a way to convert a string union type to an array, that's why this is defined this way.

Reply:
ah ok, i see now, and defining the types from the string array as const is also the way to go :)